src/lib/y2firewall/clients/auto.rb

```diff
@@ -198,6 +211,51 @@ def imported
       def imported?
         !!self.class.imported
       end
+
+      # Set of methods for formating all types of relations defined via
+      # has_many (@see Y2Firewall::Firewalld::Relations#has_many) in
+      # Y2Firewall::Firewalld::Zone
+      Firewalld::Zone.relations.each do |relation|
+        define_method("#{relation}_summary") do |zone|
+          raise ArgumentError, "zone parameter has to be defined" if zone.nil?
+
+          names = zone.send(relation)
+          return "" if names.empty?
+
+          Yast::HTML.Bold("#{relation.capitalize}:") + Yast::HTML.List(names)
+        end
+      end
+
+      # Creates a summary for the given zone
+      #
+      # @param [Firewalld::Zone] zone object defining a zone
+      # @return [String] HTML formated zone description
+      def zone_summary(zone)
+        raise ArgumentError, "zone parameter has to be defined" if zone.nil?
+
+        relations = Firewalld::Zone.relations.map(&:to_s)
+        desc = relations.map { |attr| send("#{attr}_summary", zone) }.delete_if(&:empty?)
```

Comment:
You have a clear example here of what is discussed in the other PBI

Reply:
hm, true. you got me ;-)